### ISSUE TITLE
Why is there no training and validation data?

### ISSUE THREAD

**Issue by fallcat** [UniversalDependencies/UD_Chinese-PUD#1]
Could anyone answer why there is only test data? Are the train and validation data somewhere else?

Thank you!

**Comment by dan-zeman** [comment]
This dataset was created as an additional test set for the CoNLL 2017 shared task, so there is no training data. You can run experiments in 10-fold cross-validation style, or you can train your models on [the other Chinese treebank](https://github.com/UniversalDependencies/UD_Chinese-GSD).
